tests/email/test_dm_notify.py

```diff
@@ -13,12 +13,14 @@
 FIXTURES_DIR = os.path.join(os.path.dirname(__file__), 'fixtures')
 
 
+_test_api_key = '1111111111' * 8
```

Comment:
`'1' * 80` might be a tad simpler?

Reply:
*shrug* wasn't my line, just transplanted it